tools/importer/transformers/links.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * Prefixes relative links with the target domain and remove trailing slash.
+ */
+export default function process(main, document) {
+  document.querySelectorAll('a').forEach((a) => {
+    const targetDomain = 'https://main--sazerac-wheatleyvodka--hlxsites.hlx.page';
+    // if the link is relative, make it absolute
+    if (a.href.startsWith('/')) {
+      let link = a.href;
+      const p1 = a.href.indexOf('#');
+      const p2 = a.href.indexOf('?');
+      let p = p1;
+      if (p1 < 0 || (p2 > 0 && p2 < p1)) {
+        p = p2;
+      }
+      if (p > 0) {
```

Comment:
can you please add some comments an what you do here? If I understood correctly, you look for URL params and reapply. those after you removed a trailing slash, if any.
maybe add an example URL and show what p1,p2 is.